script.js

```diff
@@ -0,0 +1,94 @@
+const formSubmit = document.querySelector('.submitBtn');
+const { forms } = document;
+const bookList = document.querySelector('.book-list');
+let bookStore = [];
+
+class UserInterface {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = new Date().valueOf();
+  }
+
+  static updateBrowser(book) {
+    const bookDiv = document.createElement('div');
+    bookDiv.classList.add('book-holder');
+    // add book title
+    const titleP = document.createElement('p');
+    titleP.textContent = book.title;
+    // add author
+    const authorP = document.createElement('p');
+    authorP.textContent = book.author;
+    // add hidden id field
+    const bookId = document.createElement('p');
+    bookId.setAttribute('type', 'hidden');
+    bookId.setAttribute('value', book.id);
+    //  create delete button
+    const deleteBtn = document.createElement('button');
```

Comment:
- [ ] I would recommend using CSS styles to make the application match [this wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). Please add the styling.